android/java/org/chromium/chrome/browser/BraveRewardsExternalWallet.java

```diff
@@ -116,33 +101,21 @@ public String toString() {
                 + '\'' + ", mLoginUrl='" + mLoginUrl + '\'' + '}';
     }
 
-    public static String WalletStatusToString (@WalletStatus int status){
+    public static String WalletStatusToString(int status) {
```

Comment:
Curious: what does the generated Java code for `WalletStatus` look like? Isn't it a strongly-typed enum?

Reply:
It's not the direct string conversion from the enum. It's mainly for debug purpouses.

Reply:
WalletStatus is integer in java. and that's why we are passing value directly. So WalletStatus.DISCONNECTED_VERIFIED would give 4 as a value.

Reply:
Wow, do Mojom folks really generate `int`s from `WalletStatus`? That's too bad. I thought it's something like:
```java
public enum WalletStatus { NOT_CONNECTED, CONNECTED, LOGGED_OUT }
```